lib/Web/Server.hs

```diff
@@ -69,11 +70,12 @@ throwCofferServerErrors err codesAndMsgs = do
   let cofferServerErrors = uncurry (flip CofferServerError) <$> codesAndMsgs
   throwError $ addApplicationJSONHeader err { errBody = encode cofferServerErrors }
 
-handleViewResult :: ViewResult -> Handler ViewResult
+handleViewResult :: ViewResult -> Handler Directory
 handleViewResult = \case
-  res@VRDirectory{} -> pure res
-  res@VREntry{} -> pure res
-  res@VRField{} -> pure res
+  VRDirectory dir -> pure dir
+  VREntry entry -> pure $ singleton entry
+  VRField{} ->
+    throwError $ err500 { errBody = "Unexpected VRField" }
```

Comment:
Same suggestion as in [this comment](https://github.com/serokell/coffer/pull/99#discussion_r878303809), use `throwCofferServerError` with an error code of 0.

Reply:
Done